### PR TITLE
Firmware preset to configure MSP-OSD for use with DJI HD Digital

### PR DESCRIPTION
**Pull request by benlumley**
Hopefully the title explains it...

### CODE REVIEW

**Review by limonspb — APPROVED** [review]

File: presets/4.3/vtx/DJI_FPV-WTF_MSP-OSD.txt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+#$ DESCRIPTION:
+#$ DESCRIPTION: # DJI FPV.WTF MSP-OSD setup
+#$ DESCRIPTION: Sets up MSP displayport on selected UART for use with MSP-OSD
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION: The information provided on this preset is for educational and entertainment purposes only. Betaflight makes no representations as to the safety or legality of the use of any information provided herein. End users assume all responsibility and liability for ensuring they are complying with all relevant laws and regulations.
+
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/298
+
+set vcd_video_system = PAL
```
Comment (haslinghuis) [inline presets/4.3/vtx/DJI_FPV-WTF_MSP-OSD.txt:22]: Should this be an option as some like to use NTSC?

Comment (benlumley) [reply presets/4.3/vtx/DJI_FPV-WTF_MSP-OSD.txt:22]: It's not related to NTSC or PAL; it's just about making sure the configurator can see all 16 lines to configure.

Comment (limonspb) [reply presets/4.3/vtx/DJI_FPV-WTF_MSP-OSD.txt:22]: I think for hacked DJI it should always be PAL?

Comment (benlumley) [reply presets/4.3/vtx/DJI_FPV-WTF_MSP-OSD.txt:22]: correct. 
as i say - the only reason for this is to ensure the configurator shows 16 lines (NTSC only shows 15)

Comment (haslinghuis) [reply presets/4.3/vtx/DJI_FPV-WTF_MSP-OSD.txt:22]: Thanks!

**Review by mamba2321 — COMMENTED** [review]
req change osd not showing up

**Comment by limonspb** [comment]
> req change osd not showing up

?

**Comment by Yianikk2206** [comment]
Can the preset be used on the DJI O4 lite? I have betaflight firmware 4.3 and can't see the OSD on my Googles 3.

**Comment by Symantix321** [comment]
This preset for 2025-12 only shows 7 UARTS. My DJI is on UART 8 (Flywoo Goku H743 Pro). Is there a way to add an 8th UART to the list?

**Pull request merged**